app/(tabs)/_layout.tsx

```diff
@@ -0,0 +1,40 @@
+import { Tabs, Redirect } from 'expo-router';
+import FontAwesome from '@expo/vector-icons/FontAwesome';
+import Foundation from '@expo/vector-icons/Foundation';
+import { useAuth } from '../auth/context/AuthContext';
+
+export default function TabLayout() {
+  const { user, isLoading } = useAuth();
+
+  if (isLoading) {
+    return null;
+  }
+
+  if (!user) {
+    return <Redirect href="/auth/screens/SignInScreen" />;
```

Comment:
Can we have all the urls/paths in constants that we can import anywhere?

Reply:
This is indeed a path, should we place it in a file somewhere? I'm not sure since the string would change with the location of the file and we'd still have a line of code to fix the pathing, know what I mean?

Reply:
I still think it'd be better to had this as a constant that we can adjust if needed.